justfile

```diff
@@ -37,13 +38,28 @@ init wipe="":
   echo "Install additional Go tools"
   go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
 
-  echo "Run a build to ensure go.work.sum is valid"
-  just build -vf
+  echo "Run modsync to ensure workspace is up to date"
+  just modsync
 
   echo "Ensure containers have been rebuilt"
-  just bh-dev build
+  if [[ "{{wipe}}" != "clean" ]]; then
+    just bh-dev build
+  else
+    echo "Clear volumes and rebuild without cache"
+    just bh-clear-volumes
+    just bh-clean-docker-build
+  fi
+
+  echo "Start integration testing services"
+  if [[ "{{wipe}}" == "clean" ]]; then
+    echo "Clear volumes and restart testing services without cache"
+    just bh-testing-clear-volumes
+    just bh-testing build --no-cache
+  fi
+
+  just bh-testing
```

Comment:
Consider removing this line.

I'm not huge fan of spinning up services for project initialization. IMO we should leave that to the developer to decide when those testing services should be started especially since there are devs on the team that prefer to run those services on their host machine or elsewhere.

Reply:
Fair enough, I'm fine with removing it. Goal was to reduce mental load with making sure to restart bh-testing (or know to start it at all), but I think you're right that it would be better to let devs decide on their own